libraries/node-core-library/src/FileSystem.ts

```diff
@@ -215,10 +250,18 @@ export class FileSystem {
    * @param times - The times that the object should be updated to reflect.
    */
   public static updateTimes(path: string, times: IFileSystemUpdateTimeParameters): void {
-    // eslint-disable-next-line @typescript-eslint/no-explicit-any
-    fsx.utimesSync(path, times.accessedTime as any, times.modifiedTime as any);
+    fsx.utimesSync(path, times.accessedTime, times.modifiedTime);
   }
 
+  /**
+   * An async version of {@link FileSystem.updateTimes}.
+   */
+  public static updateTimesAsync(path: string, times: IFileSystemUpdateTimeParameters): Promise<void> {
+    // The cast is required here because the fs-extra typings are incorrect.
```

Comment:
This comment seemed to beg the question of why we wouldn't just fix the typings.
```suggestion
    // This cast is needed because the fs-extra typings require both parameters
    // to have the same type (number or Date), whereas Node.js does not require that.
```